DependencyInjection/CmfMenuExtension.php

```diff
@@ -44,7 +44,9 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadExtensions($config, $loader, $container);
         }
 
-        if ($config['publish_workflow']['enabled']) {
+        if (true === $config['publish_workflow']['enabled']
+            || 'auto' === $config['publish_workflow']['enabled'] && array_key_exists('CmfCoreBundle', $bundles)
```

Comment:
i started to prefer array_key_exists because it is more strict and
reveils errors. if we refactor and don't have $bundles available
anymore, isset will happily return false, while array_key_exists tells
us that something went wrong.
performance difference is not huge, and this is the container build
phase so its not executed that much.